pkg/signature/kms/cliplugin/client.go

```diff
@@ -60,7 +60,7 @@ func LoadSignerVerifier(ctx context.Context, inputKeyResourceID string, hashFunc
 	if deadline, ok := ctx.Deadline(); ok {
 		initOptions.CtxDeadline = &deadline
 	}
-	pluginClient := newPluginClient(executable, initOptions)
+	pluginClient := newPluginClient(executable, initOptions, makeCmd)
```

Comment:
Maybe this is my confusion with go paradigms, but is there away to avoid this? Seems like we're only exposing this for testing? Do we expect some plugin client to actually provide a non-default version of this?

Reply:
I was looking around blogs and stackoverflow to see if there were any other suggestions. It seems like the only other Go-ish approach is a global variable, though I don't think that's much cleaner.